FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DNS.c

```diff
@@ -186,6 +188,8 @@ static uint32_t prvGetHostByName( const char *pcHostName,
 
 /*-----------------------------------------------------------*/
 
+/* Below #include just tells the compiler to pack the structure. 
```

Comment:
This comment seems gratuitous. Presumably, the structure packing directive is required in order to avoid hard faults in certain ports.

Reply:
Updated